platform/lib/pipeline/specImporter.js

```diff
@@ -50,12 +50,14 @@ class SpecImporter {
         doc.order = importDoc.order;
         doc.toc = importDoc.toc;
         doc.formats = importDoc.formats;
+        doc.baseURL = `https://github.com/${importDoc.repo}/blob/master/`;
```

Comment:
Since the baseURL is only used in this method there is no need to assign it to doc.
Simply define a local variable with let.